src/Api/Serializer/DiscussionSerializer.php

```diff
@@ -44,7 +44,8 @@ protected function getDefaultAttributes($discussion)
             'canReply'          => $gate->allows('reply', $discussion),
             'canRename'         => $gate->allows('rename', $discussion),
             'canDelete'         => $gate->allows('delete', $discussion),
-            'canHide'           => $gate->allows('hide', $discussion)
+            'canHide'           => $gate->allows('hide', $discussion),
+            'canViewIp'         => $gate->allows('viewIpOfPost', $discussion)
```

Comment:
Can you add a trailing slash here, please? (And in `PostSerializer`, too.)

I know they weren't they before, but I generally consider this to be a good practice.